userdata.tmpl

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+add-apt-repository -y ppa:wireguard/wireguard
+apt-get update -y
+apt-get upgrade -y
```

Comment:
Suggestion: You might comment this out, just to speed things up a bit while developing. Upgrading all installed packages will probably take a good bit of time.